src/unitxt/inference.py

```diff
@@ -107,6 +114,10 @@ def _set_inference_parameters(self):
                 if param_inst_val is None:
                     setattr(self, param, param_dict_val)
 
+    def get_model_details(self) -> Dict:
```

Comment:
I think this should return an empty dict or None (And the the return type should be Optional[Dict]).
This way if someone call the method and it was nt implemented, no bug will happen